### ISSUE TITLE
Ability to change the file output name with custom date formatting in the config file

### ISSUE THREAD

**Issue by brodi1** [jtheoof/swappy#62]
Example:
```

date_format=%Y-%m-%d_%H-%M-%S-%N

file_output_name=Swappshot-$date_format.png
```

Filename: `Swappshot-2020-11-09_17-12-56-924287207.png`

**Comment by fabean** [comment]
Also, would like this for some reason the current format stops me from uploading to Jira without error.

**Comment by Isti115** [comment]
Sadly MS Teams doesn't accept the generated files either (Took me a while to figure out that the reason is their name...), so I always have to rename them before I can send them there. A format option would be awesome!

**Comment by apiraino** [comment]
@fabean @Isti115 I'm curious about the issue you're experiecing.

Provided that my patch should "fix" your issues, I believe the *real* issue is on those platforms because (imo) `swappy-stdin-XXXXXX.png Thu Dec  3 10:39:36 2020.png` is a perfectly valid filename to upload via a form.

I can only speculate that the semicolon `:` is causing the problem and changing the filename will simply sweep under the rug an issue on their end.

**Comment by Isti115** [comment]
Yes, I'm absolutely sure that the problem is on Microsoft's side, the Teams experience is sometimes borderline horrible under linux (I've already had the desktop app crash during a class that I was teaching and straight up refusing to reopen, so I had to continue from Chrome and have had other issues as well, such as for example my microphone randomly stops working, which is not a configuration issue as far as I can tell, since it works flawlessly in Discord...), but due to the constraints of the university I have to keep using it and since MS doesn't seem to be too keen on fixing these things I sadly have to resort to adapting to them instead.

**Comment by apiraino** [comment]
Fixed in #64
